extensions/azuremonitor/src/extension.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { SqlOpsDataClient, ClientOptions } from 'dataprotocol-client';
+import { ServerOptions, TransportKind } from 'vscode-languageclient';
+import { localize } from './localization';
+import * as Constants from './constants';
+import * as Strings from './strings';
+import { output } from './ui-references';
+import { ClientErrorHandler } from './client-error-handler';
+import { SerializationFeature } from './features/serializationFeature';
+
+let isActivated = false;
+
+export function activate(context: vscode.ExtensionContext): void {
+	if (isActivated) {
+		return;
+	}
+
+	output.appendLine(
+		localize("extension.activating", "Activating {0}.", Strings.extensionName));
+
+	launchServiceClient(path.join(__dirname, '../sqltoolsservice/windows/3.0.0-release.1/MicrosoftKustoServiceLayer.exe'), context);
```

Comment:
I don't see the service downloader config for this, where are you getting STS from?

Reply:
I haven't done that effort yet because I'm waiting for it to get merged and deployed to blob storage.